tools/acrn-crashlog/acrnprobe/Makefile

```diff
@@ -0,0 +1,73 @@
+MAJOR_VERSION=1
+MINOR_VERSION=0
+
+BASEDIR 	:= $(shell pwd)
+
+LIBS		= -lpthread -lxml2 -lcrypto -lrt -lsystemd -ltelemetry
```

Comment:
What is the Clearlinux bundle that provides the `telemetry` dev files?

Can you also indicate which extra packages should be installed on Fedora and Ubuntu in order to be able to compile these tools? (we'll want to add those to the Getting Started Guide)

Reply:
Currently, there is no telemetry-dev bundle. We rpm install the dev package on clearlinux.

Now we're trying to provide a patch to cover the env issues.
Except the telemetrics packages, we need following lib on these OS:
Ubuntu 14.04:
libsystemd-journal-dev
libevent-dev
libssl-dev
libxml2-dev                   

Ubuntu 16.04:
libsystemd-dev
libevent-dev
libssl-dev
libxml2-dev                   

ClearLinux:
libsystemd-dev
libevent-dev
libopenssl-dev
libxml2-dev

Reply:
> ClearLinux:
> libsystemd-dev
> libevent-dev
> libopenssl-dev
> libxml2-dev

All these dev dependencies can be installed via `os-core-dev` and `os-core-update-dev`. The only one missing is the `telemetry` piece. @miguelinux , I couldn't find any Clearlinux bundle that includes the dev files for telemetry. Could we define one? Without this, there is no easy way at the moment to build these new ACRN tools on Clearlinux.

Reply:
Hi @chengangc is there a way to easily compile the tools without using the `telemetry` piece? If so, how would that affect the usability of the tools?

Reply:
Hi @gvancuts This tool works as a probe of telemetrics-client. So we can't compile the tools without telemetry. Currently, we get the dev files with installing the dev rpm(https://download.clearlinux.org/current/x86_64/os/Packages/telemetrics-client-dev-1.16.0-93.x86_64.rpm).

Reply:
This is really ugly, we're going to have to tell everyone to install an `rpm` file on a Clearlinux system to compile the tools while RPMs are not officially supported and things should be installed via bundles. @miguelinux @bryteise , could we integrate the appropriate dev files (for the `telemetry` package) in a bundle so we don't have to use this 'trick'? What would be the appropriate bundle to modify to add this, [os-clr-on-clr](https://github.com/clearlinux/clr-bundles/blob/master/bundles/os-clr-on-clr), ...?

Reply:
Yes, put it in os-clr-on-clr please.

Reply:
Is this how it should be done? https://github.com/clearlinux/clr-bundles/pull/48

(I've also seen more patches being sent on the clr-dev mailing list but I'm not sure how to properly format those myself).

Reply:
I'll put a review there with an explanation.